pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -188,12 +188,21 @@ func (s *Reconciler) Update(ctx context.Context) error {
 	}
 
 	if vm.NetworkProfile != nil && vm.NetworkProfile.NetworkInterfaces != nil {
+		var vnetName string
```

Comment:
We have this same block 3 times - turn it into a utility function?

Reply:
This will get removed once the installer sets both new fields.